addons/hr_timesheet/report/report_timesheet_templates.xml

```diff
@@ -27,11 +27,14 @@
                                <span t-if="line.user_id.partner_id.name" t-field="line.user_id.partner_id.name">Audrey Peterson</span>
                                <span t-else="" t-field="line.employee_id">Audrey Peterson</span>
                             </td>
-                            <td t-if="show_project" class="align-middle">
-                                <span t-field="line.project_id.sudo().name">Research and Development</span>
-                            </td>
-                            <td t-if="show_task" class="align-middle">
-                                <span t-if="line.task_id" t-field="line.task_id.sudo().name">New Portal System</span>
+                            <t t-set="timesheet_record_info" t-value="False"/>
+                            <t t-if="show_project">
+                                <t t-set="timesheet_record_info" t-value="line.project_id.sudo().name"/>
+                                <t t-if="show_task and line.task_id" t-set="timesheet_record_info" t-value="'%s / %s' % (timesheet_record_info, line.task_id.sudo().name)"/>
```

Comment:
Since we're doing string interpolation, would an f-string not be cleaner?

Reply:
Hello @MissingNoShiny 
f-string is not supported in xml. any alternative?

Thank You :)